app/models/academic_year.rb

```diff
@@ -80,4 +95,15 @@ def -(other)
   def +(other)
     AcademicYear.new(start_year + other)
   end
+
+  private
+
+  def years=(start_year)
+    if start_year.nil?
+      @start_year = @end_year = nil
+    else
+      @start_year = start_year.nil? ? nil : start_year.to_s.split("/").first.to_i
```

Comment:
@nomasprime My sense is that start_year.nil? has been covered in the if branch. Have I missed understood something in your proposed change?

Reply:
Good spot. Think this got used in different context and hasn't been refactored how it should have been.